src/lancer.ts

```diff
@@ -107,6 +107,14 @@ import handleSocketMessage from "./module/socket";
 
 const lp = LANCER.log_prefix;
 
+declare module "fvtt-types/configuration" {
+  namespace Hooks {
+    interface ApplicationConfig {
+      CombatDock: Application.Any;
+    }
```

Comment:
I don't know where `CombatDock` comes from. However the system expected some application hooks to exist for it. The way for fvtt-types to automatically register your hooks is this.